src/app/games/lasca/LascaTutorial.ts

```diff
@@ -0,0 +1,78 @@
+            LascaState.getInitialState(),
+        ),
+        // 3. Simple step
+        TutorialStep.anyMove(
+            $localize`Steps`,
+            $localize`A simple step is made by one diagonal move forward, left or right. Click on the chosen piece, then on it's landing square.<br/><br/>You are playing Dark, do the first move.`,
+            LascaState.getInitialState(),
+            LascaMove.fromStep(new Coord(4, 4), new Coord(3, 3)).get(),
+            $localize`Congratulations!`,
+        ),
+        // 4. Simple capture
+        TutorialStep.anyMove(
+            $localize`Capture`,
+            $localize`You have to capture when you can. It is the case here, so click on the piece that must capture, and then on its landing square.<br/><br/>You're playing Dark, go ahead.`,
+            LascaState.from([
+                [_v, __, __, __, _v, __, _v],
+                [__, __, __, _v, __, _v, __],
+                [__, __, _v, __, _v, __, _v],
+                [__, _v, __, __, __, __, __],
+                [_u, __, _u, __, _u, __, _u],
+                [__, _u, __, _u, __, _u, __],
+                [_u, __, _u, __, _u, __, _u],
+            ], 2).get(),
+            LascaMove.fromCapture([new Coord(2, 4), new Coord(0, 2)]).get(),
+            $localize`Congratulations, notice that the captured piece was not removed from the board, but put below the capturing pieces.`,
+        ),
+        // 5. Longer capture
+        TutorialStep.anyMove(
+            $localize`Multiple captures (1/2)`,
+            $localize`If, after the beginning of your capture, the piece that just moved can capture another piece, it has to capture until she can no longer. To do so, you must then click again on the next landing space. Note that, when capturing with an officer, you cannot jump twice over the same stack.<br/><br/>You are playing Dark, a double capture is possible, go ahead.`,
+            LascaState.from([
+                [_v, __, __, __, _v, __, _v],
+                [__, _v, __, _v, __, _v, __],
+                [__, __, _v, __, _v, __, _v],
+                [__, _v, __, __, __, __, __],
+                [__, __, _u, __, _u, __, _u],
+                [__, __, __, _u, __, _u, __],
+                [_u, __, __, __, _u, __, _u],
+            ], 2).get(),
+            LascaMove.fromCapture([new Coord(2, 4), new Coord(0, 2), new Coord(2, 0)]).get(),
+            $localize`Congratulations, you captured two pieces, and, by reaching the last line, your piece has now been promoted to the rank of officer ! Now, it can move and capture backward, which is illegal for normal piece.`,
+        ),
+        TutorialStep.informational(
+            $localize`Minoritary capture is allowed`,
+            $localize`If you have several capture choices, one capturing one piece, one capture two, you are allowed to choose either.`,
```

Comment:
one capture two?!

Reply:
ça aidera pas si la première phrase ajoute pas d'info, ils savent déjà qu'ils ont le choix entre leurs choix, ce serait comme leur dire "vous avez le droit de jouer" :D

Reply:
Justement, c'est pour différencier des dames non ? Où si tu as plusieurs choix, t'as pas le choix :D

Reply:
Je comprends plus le soucis de ma phrase, go plusieurs propositions car justement l'objectif içi c'est 1. décrire la situation 2. dire les choix qui sont légaux, la différence d'un tuto de dames serait dans le deux ont dirais pas "do whatever" mais "do the longest"
Où sommes nous debout à propos ça ?

Reply:
Proposition:
`If you have several capture choices, you are allowed to choose any of them. For example if one choice is to capture one piece, and the other choice is to capture two pieces, you can choose either.`

Le soucis c'est d'énoncer une règle générale en la mélangeant a un exemple. Séparer les deux (règle générale, exemple) évite confusion possible.

Reply:
Done !